sql/2021/ecommerce/pagestats_html_bydevice.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 13_07: Distribution of HTML kilobytes per page
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(APPROX_QUANTILES(bytesHtml, 1000)[OFFSET(100)] / 1024, 2) AS p10,
```

Comment:
2019 queries did rounding in the SQL but we've since changed to prefer decimal values that get formatted in Sheets. Update here and throughout.

See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide#formatting-results for more info.
```suggestion
 APPROX_QUANTILES(bytesHtml, 1000)[OFFSET(100)] / 1024 AS p10,
```

Reply:
I fixed this as well for this particular query. Let me audit the rest of the queries and fix those as well to match current best practices

Reply:
I have fixed it for the other queries as well